Core/Dialog/PSPGamedataInstallDialog.cpp

```diff
@@ -265,7 +271,7 @@ void PSPGamedataInstallDialog::DoState(PointerWrap &p) {
 		param.ptr = 0;
 	}
 
-	if (s > 3) {
```

Comment:
Oh, this is the bug - it was previously at version 3 but this part would never execute.

Reply:
Right.  And because rewind runs WRITE every so often, it'd run the else portion and reset progress.

-[Unknown]